### ISSUE TITLE
WCAG 1.3.1 A: List not marked up (Issue 8)	

### ISSUE THREAD

**Issue by CJE001** [civictheme/uikit#384]
### Summary

Via Vision Australia assessment: August 2024

Impact: low

Note: DTA have a 90 day remediation period to address the identified issues within the audit, all issues must be resolved to obtain WCAG 2.2 certification for digital.gov.au.

### Steps to reproduce

[digital.gov.au beta | digital.gov.au](https://www.digital.gov.au/) (https://www.digital.gov.au)

### Observed outcome

The linked text "Privacy", "Disclaimer and copyright" and "Accessibility" in the sitewide footer has been visually styled as and is structurally a list; however, this text has not been marked up semantically in the code as a list. 

![Picture6](https://github.com/user-attachments/assets/06a9d8f3-58a2-4e67-a353-160ccef37f68)

### Code used

```
<p class="text-align-right ct-text-small">
<a href="/privacy-statement" data-entity-type="node" data-entity-uuid="1445fbb1-cd8e-42b4-b619-3686e3526d53" data-entity-substitution="canonical" class="ct-content-link ct-theme-dark" title="Privacy statement">
Privacy
</a>
l 
<a href="/disclaimer-and-copyright" data-entity-type="node" data-entity-uuid="a0536d3d-905d-40d7-b72d-036ce92f2dd6" data-entity-substitution="canonical" class="ct-content-link ct-theme-dark" title="Disclaimer and copyright">
Disclaimer and copyright
</a>
l 
<a href="/accessibility-statement" data-entity-type="node" data-entity-uuid="5b4a3a14-45ce-4e62-9dae-a2436bd61b34" data-entity-substitution="canonical" class="ct-content-link ct-theme-dark" title="Accessibility statement">
Accessibility
</a>
</p>
```
### Why this matters

Lists provide important information about content and relationships. When a screen reader encounters a list, it will announce to users how many items are within the list, and as a user navigates the list items, information on the type of list. Screen reader users can also use shortcuts to skip between lists and list items on a screen.

When text functions as a lists but lacks the proper semantic markup, this information is not available to assistive technologies. Consequently, screen reader users may not realise the content is related or the relationship between list items. Screen reader shortcuts to navigate via lists will not be available to users. 

### Expected outcome

Ensure lists are marked up using semantic list markup.
